pkgs/development/libraries/faad2/default.nix

```diff
@@ -13,19 +13,42 @@
 
 stdenv.mkDerivation rec {
   pname = "faad2";
-  version = "2.10.1";
+  version = "2.11.0";
 
   src = fetchFromGitHub {
     owner = "knik0";
     repo = "faad2";
     rev = version;
-    sha256 = "sha256-k7y12OwCn3YkNZY9Ov5Y9EQtlrZh6oFUzM27JDR960w=";
+    hash = "sha256-fp8RGG9HQOcyudTA9osGaI3oVIY8vo8vhJtd04+wNKc=";
   };
 
-  configureFlags = []
-    ++ lib.optional drmSupport "--with-drm";
+  patches = [
```

Comment:
btw, [faad2 2.11.1](https://github.com/knik0/faad2/releases/tag/2.11.1) was released a few hours ago, and it includes these patches and builds the shared library by default

Reply:
Good point. Proposed an update as https://github.com/NixOS/nixpkgs/pull/267515